apps/YellowSensorToCloud/components/sensors/light/Component.cdef

```diff
@@ -6,43 +6,40 @@
 //--------------------------------------------------------------------------------------------------
 cflags:
 {
-    -I$CURDIR/../../fileUtils
-    -I$LEGATO_ROOT/apps/sample/dataHub/components/periodicSensor
- //   -I${MANGOH_ROOT}/components/libiioComponent/libiio
     -std=c99
 }
 
 provides:
 {
     api:
     {
-        $CURDIR/../../../interfaces/light.api
+        light.api
     }
 }
 
 requires:
 {
     api:
     {
-        dhubIO = io.api
+        dhubIO = io.api [types-only]
     }
 
     component:
     {
         ../../fileUtils
-        $LEGATO_ROOT/apps/sample/dataHub/components/periodicSensor
-//        ${MANGOH_ROOT}/components/libiioComponent
+        periodicSensor
+//        libiioComponent
     }
     file:
     {
 #if ${LEGATO_TARGET} = wp85
         /sys/bus/i2c/devices/3-0044/iio:device2/in_illuminance_input     /driver/
- 
+
 #elif ${LEGATO_TARGET} = wp750x
        /sys/bus/i2c/devices/3-0044/iio:device2/in_illuminance_input     /driver/
 
 #elif ${LEGATO_TARGET} = wp76xx
-       /sys/bus/i2c/devices/7-0044/iio:device2/in_illuminance_input     /driver/
+       /sys/bus/i2c/devices/7-0044/iio:device1/in_illuminance_input     /driver/
```

Comment:
Are the iio devices values based on BMM connected to BMI or to WP ?

Reply:
This would be based on the BMM connected to the BMI.
I'm guessing that's the root cause of why I had to change it.  The reason I changed it is because it was wrong. :)

Reply:
It could very well also be wrong on other modules. I have not tried those. I only have a WP7603.

Reply:
Given that we have made the decision to drop wp75/85 for the mangOH Yellow, we don't need and `#if/#else` at all.  I think Ashish is probably correct that this is related to the positioning of the BMM.  as a result, I think we should replace the whole chunk with:
```
/sys/bus/i2c/devices/7-0044/iio:device1/in_illuminance_input     /driver/
```